executor/show_test.go

```diff
@@ -182,7 +182,29 @@ func (s *testSuite2) TestIssue18878(c *C) {
 	tk.MustQuery("show grants for `show_grants`@`127.0.%`")
 }
 
+<<<<<<< HEAD
```

Comment:
Please resolve conflicts. @imtbkcat

Reply:
It seems this problem only happen in 4.0 version, so this pr will be closed.